pixart.js

```diff
@@ -1 +1,16 @@
+var setColors =document.querySelector(".color");
+var colorField =document.querySelector(".square");
+var brush = document.querySelector(".brush");
 
+$("button")[0].addEventlistener("clicks", handleClickEvent);
```

Comment:
This code requires the jQuery library.  We will be covering that soon.  You have already created variable, above, that have a reference to the objects you want on the page (in the DOM).  I recommend you use them.